specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -40,7 +40,7 @@
           "UsageDetails"
         ],
         "operationId": "UsageDetails_List",
-        "description": "Lists the usage details for the defined scope. Usage details are available via this API only for May 1, 2014 or later.",
+        "description": "Lists the usage details for the defined scope. Usage details are available via this API only for May 1, 2014 or later. For more information on using this API, including how to specify a date range, please see: https://docs.microsoft.com/en-us/azure/cost-management-billing/costs/manage-automation",
```

Comment:
Should we use this link will takes them to the exact sub heading https://docs.microsoft.com/en-us/azure/cost-management-billing/costs/manage-automation#get-usage-details-for-a-scope-during-specific-date-range

Reply:
I considered this but decided against it. The target page is not very large and there is other useful information there. The main issue is that people don't know about the page and this addresses that.